lms/config/__init__.py

```diff
@@ -72,6 +72,7 @@ def configure(settings):
         "admin_auth_google_client_secret": sg.get("ADMIN_AUTH_GOOGLE_CLIENT_SECRET"),
         "blackboard_api_client_id": sg.get("BLACKBOARD_API_CLIENT_ID"),
         "blackboard_api_client_secret": sg.get("BLACKBOARD_API_CLIENT_SECRET"),
+        "jstor_ip": sg.get("JSTOR_IP"),
```

Comment:
Ultimately the IP (or other credentials) will need to be associated either with the LMS app installation directly or indirectly (eg. if we make it a property of an entity of the institution/school associated with the installation). For an initial internal demo that is only usable by the Hypothesis test LMS installations, we could use a fixed IP in an env var.